packages/serverless-api/src/client.ts

```diff
@@ -150,6 +161,125 @@ export class TwilioServerlessApiClient extends events.EventEmitter {
     return this.client;
   }
 
+  async setEnvironmentVariables(
+    config: SetEnvironmentVariablesConfig
+  ): Promise<SetEnvironmentVariablesResult> {
+    let serviceSid: Sid | undefined = config.serviceSid;
+    if (
+      typeof serviceSid === 'undefined' &&
+      typeof config.serviceName !== 'undefined'
+    ) {
+      serviceSid = await findServiceSid(config.serviceName, this);
+    }
+
+    if (typeof serviceSid === 'undefined') {
+      throw new Error('Missing service SID argument');
+    }
+
+    let environmentSid;
+
+    if (!isEnvironmentSid(config.environment)) {
+      const environmentResource = await getEnvironmentFromSuffix(
+        config.environment,
+        serviceSid,
+        this
+      );
+      environmentSid = environmentResource.sid;
+    } else {
+      environmentSid = config.environment;
+    }
+
+    return { serviceSid, environmentSid };
+  }
```

Comment:
I know this is a draft PR, but this function is called `setEnvironmentVariables` but it only seems to collect a `serviceSid` and `environmentSid` and return them. Think it's missing the bit where it sets the vars! The `get` and `remove` functions do carry out their purpose, so I thought I'd call this one out.

Reply:
Ha yeah it's not implemented yet 😄